### PR TITLE
add weight_decay arguments

### PR DESCRIPTION
**Pull request by ouyangyu**
参考：https://github.com/tensorflow/addons/blob/v0.12.0/tensorflow_addons/optimizers/lamb.py#L44-L46
https://github.com/Oneflow-Inc/oneflow/blob/master/oneflow/python/ops/optimizer.py#L1423-L1425

### CODE REVIEW

File: oneflow/python/test/ops/test_lamb.py [hunk 1]
```diff
@@ -56,6 +64,7 @@ def testAdam(
                 beta1=beta1,
                 beta2=beta2,
                 epsilon=epsilon,
+                weight_decay=weight_decay,
```
Comment (ScXfjiang) [inline oneflow/python/test/ops/test_lamb.py:67]: 这里的job之前命名为testAdam，这个PR顺带修改下

File: oneflow/python/ops/optimizer.py [hunk 2]
```diff
@@ -1736,6 +1736,9 @@ class LAMB(Optimizer):
         beta2 (float, optional): The exponential weighted average decay rate for the 2rd-moment estimates (:math:`\beta_2`). Defaults to 0.999.
         epsilon ([type], optional): A small float constant value for numerical stability (:math:`\epsilon`). Defaults to 1e-6.
         loss_scale_factor (Optional[float], optional): The scale factor of loss. Defaults to None.
+        weight_decay (Optional[float], optional): The weight decay factor (In the equation is :math:`\lambda`). Defaults to None.
+        weight_decay_includes (Optional[Union[Sequence[Text], Text]], optional): The name of the model parameters that use weight decay. Defaults to None.
+        weight_decay_excludes (Optional[Union[Sequence[Text], Text]], optional): The name of the model parameters that do not use weight decay. Defaults to None.
```
Comment (ScXfjiang) [inline oneflow/python/ops/optimizer.py:1741]: 像其他带有weight decay的Optimizer一样加入注释说明参数限制吧：
> Only one of `weight_decay_includes` and `weight_decay_excludes` can be set. If both are None, all the model parameters will use weight decay.

**Review by ScXfjiang — APPROVED** [review]
